unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Utils/UserPositionMarkerUtils.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+internal class ExclusionArea
```

Comment:
Could we add a documentation header here to explain what is the responsibility of this class? Thanks!